data/sql/updates/pending_db_world/rev_1673176236867407100.sql

```diff
@@ -0,0 +1,7 @@
+--
+UPDATE `creature_template` SET `AiName`='', `ScriptName`='npc_underbat' WHERE `entry`=17724;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=17724 AND `source_type`=0;
+
+DELETE FROM `spelldifficulty_dbc` WHERE `ID`=34171;
+INSERT INTO `spelldifficulty_dbc` VALUES
+(34171,34171,37956,0,0);
```

Comment:
why are we changing dbc values?

Reply:
We are not changing dbc values - we are adding missing entries.

Reply:
I am researching and checking dbcs from 243 to 10.1 and i do not see 34171 ever exsisting in the spell difficulties dbc/db2 files ever.

Reply:
That's why I'm adding.